### PR TITLE
[doc] typo fixes

### PR DESCRIPTION
**Pull request by keonkim**

### CODE REVIEW

**Comment by soumith** [comment]
Thanks @keonkim

File: doc/random.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ random number generator.
 ## Seed Handling ##
 
 The random number generator is provided with a random seed via
-[seed()](#torch.seed) when torch is being initialised. It can be
+[seed()](#torch.seed) when torch is being initialized. It can be
```
Comment (Atcold) [inline doc/random.md]: LOL... Torch has a French author and British spelling :stuck_out_tongue:

File: doc/random.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Returns an unsigned 32 bit integer random number.
 <a name="torch.uniform"></a>
 ### [number] uniform([a],[b]) ###
 
-Returns a random real number according to uniform distribution on [a,b[. By default `a` is 0 and `b` is 1.
+Returns a random real number according to uniform distribution on [a,b]. By default `a` is 0 and `b` is 1.
```
Comment (bamos) [inline doc/random.md]: Hi, the original `[a,b[` here is correct as @fmassa pointed out in https://github.com/torch/torch7/pull/427. [The code](https://github.com/torch/torch7/blob/master/lib/TH/THRandom.c#L203) shows it's an open interval in `b`.

Is `[a,b)` clearer than `[a,b[` here? I haven't seen the reversed-bracket notation for open intervals before, and it seems like this is confusing other people as well.

Comment (keonkim) [inline doc/random.md]: Oh, ok. I think `[a,b)` gives better intuition

Comment (Atcold) [inline doc/random.md]: `[a, b[` and `[a, b)` are equivalent. The first is used by physicists, the second by mathematicians.

**Pull request merged**